website/docs/r/lambda_permission.html.markdown

```diff
@@ -141,16 +140,16 @@ resource "aws_lambda_permission" "lambda_permission" {
  	The permission will then apply to the specific qualified ARN.
  	e.g. `arn:aws:lambda:aws-region:acct-id:function:function-name:2`
  * `source_account` - (Optional) This parameter is used for S3 and SES. The AWS account ID (without a hyphen) of the source owner.
- * `source_arn` - (Optional) When granting Amazon S3 or CloudWatch Events permission to
- 	invoke your function, you should specify this field with the Amazon Resource Name (ARN)
- 	for the S3 Bucket or CloudWatch Events Rule as its value.  This ensures that only events
- 	generated from the specified bucket or rule can invoke the function.
- 	API Gateway ARNs have a unique structure described
- 	[here](http://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-control-access-using-iam-policies-to-invoke-api.html).
+ * `source_arn` - (Optional) When the principle is an AWS service, the ARN of the specific resource within that service to grant permission to.
+  Without this, any resource from `principle` will be granted permission – even if that resource is from another account.
+  For S3, this should be the ARN of the S3 Bucket.
+  For CloudWatch Events, this should be the ARN of the CloudWatch Events Rule.
+  For API Gateway, this should be the ARN of the API, as described [here][2].
  * `statement_id` - (Optional) A unique statement identifier. By default generated by Terraform.
  * `statement_id_prefix` - (Optional) A statement identifier prefix. Terraform will generate a unique suffix. Conflicts with `statement_id`.
 
 [1]: https://developer.amazon.com/docs/custom-skills/host-a-custom-skill-as-an-aws-lambda-function.html#use-aws-cli
+[2]: http://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-control-access-using-iam-policies-to-invoke-api.html.
```

Comment:
link unfortunately doesn't open correctly with the ending `.` but if removed, works as expected. ah also do you mind changing this to `https`? will be helpful for when AWS officially decides to no longer accept `http` for this page